includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -181,11 +177,10 @@ private function getLangWithFallback( $lang ) {
 			'user-register' => 'https://www.wikia.com/register?uselang=ja',
 		],
 		'pl' => [
-			'explore-wikis' => 'http://fandom.wikia.com/explore?uselang=pl',
-			'fandom-logo' => 'http://pl.wikia.com',
+			'explore-wikis' => 'http://pl.wikia.com/Wikia',
```

Comment:
They were to stay without changes, so I left them as they are now. Those links will be removed (or updated) soon, anyway.